builtin/logical/database/path_roles.go

```diff
@@ -522,11 +522,11 @@ func (b *databaseBackend) pathStaticRoleCreateUpdate(ctx context.Context, req *l
 }
 
 type roleEntry struct {
-	DBName        string              `json:"db_name"`
-	Statements    dbplugin.Statements `json:"statements"`
-	DefaultTTL    time.Duration       `json:"default_ttl"`
-	MaxTTL        time.Duration       `json:"max_ttl"`
-	StaticAccount *staticAccount      `json:"static_account" mapstructure:"static_account"`
+	DBName        string         `json:"db_name"`
+	Statements    v4.Statements  `json:"statements"`
```

Comment:
Not specific to this PR, but noticed that we're still using a v4 struct for storing the statements even though there are v5 equivalents of these. We'll probably have to handle upgrades if we wanted to fully migrate out of v4.

Reply:
We can certainly look into it, though I suspect we'll end up keeping the same or very similar struct definition for these since the list of statements hasn't changed and still needs to be stored.